docs/source/usersguide/processing.rst

```diff
@@ -34,30 +34,20 @@ as requested; it is used in many of the provided plotting utilities, OpenMC's
 regression test suite, and can be used in user-created scripts to carry out
 manipulations of the data.
 
-An `example notebook <../examples/post-processing.ipynb>`_ demonstrates how to
-extract data from a statepoint using the Python API.
+An `example notebook`_ demonstrates how to extract data from a statepoint using
+the Python API.
 
 Plotting in 2D
 --------------
 
-The `notebook example <../examples/post-processing.ipynb>`_ also demonstrates
-how to plot a structured mesh tally in two dimensions using the Python API. One
-can also use the :ref:`scripts_plot` script which provides an interactive GUI to
-explore and plot structured mesh tallies for any scores and filter bins.
+The `example notebook`_ also demonstrates how to plot a structured mesh tally in
+two dimensions using the Python API. One can also use the :ref:`scripts_plot`
+script which provides an interactive GUI to explore and plot structured mesh
+tallies for any scores and filter bins.
 
 .. image:: ../_images/plotmeshtally.png
    :width: 400px
 
-Getting Data into MATLAB
```

Comment:
Was this section no longer relevant/useful?

Reply:
I personally don't recommend people to use MATLAB given the capabilities available in Python and its open source ecosystem. If someone really thinks we ought to leave this in though, now is the time to speak up.